src/components/CollectionItem.astro

```diff
@@ -37,9 +37,6 @@ const hasDescription = !!description?.trim();
       <SectionHeader headingLevel="h2" class="usa-collection__heading font-body-lg">
         <Link href={link}>{title}</Link>
       </SectionHeader>
-      <ul class="usa-collection__meta" aria-label="More information">
```

Comment:
You're failing some tests with the removal of this `ul` element. Did you plan on removing the date from the CollectionItem?